storage/libradb/src/ledger_store/mod.rs

```diff
@@ -94,7 +94,8 @@ impl LedgerStore {
     /// Return the ledger infos reflecting epoch bumps with their least 2f+1 signatures starting
     /// from `start_epoch` to the most recent one.
     /// Note: ledger infos and signatures are available at the last version of each earlier epoch
-    /// and at the latest version of current epoch, we filter out the latter.
+    /// and at the latest version of current epoch.
+    /// We filter out the one that're equal or higher than `ledger_version`.
```

Comment:
The old comment looks a little better.

Reply:
lol we're not only filtering the latter now, we're filtering the first if the last one has the same version as `ledger_version`

Reply:
okay i understand now